tests/upgrades/test_yum_plugins.py

```diff
@@ -232,42 +198,35 @@ def test_pre_scenario_yum_plugins_count(self):
             1. The content host is created.
             2. katello-agent install and goferd run.
         """
-        org = entities.Organization().create()
-        loc = entities.Location(organization=[org]).create()
-
-        environment = entities.LifecycleEnvironment(organization=org
+        environment = entities.LifecycleEnvironment(organization=self.org
                                                     ).search(query={'search': 'name=Library'})[0]
-        repos = self._create_rh_rhel_tools_repos(org.id)
-        content_view = self._publish_content_view(org=org, repolist=repos)
+        repos = self._get_rh_rhel_tools_repos()
+        content_view = self._publish_content_view(org=self.org, repolist=repos)
         ak = entities.ActivationKey(content_view=content_view,
-                                    organization=org.id,
+                                    organization=self.org.id,
                                     environment=environment).create()
 
-        rhel7_client = dockerize(ak_name=ak.name, distro='rhel7', org_label=org.label)
+        rhel7_client = dockerize(ak_name=ak.name, distro='rhel7', org_label=self.org.label)
         client_container_id = [value for value in rhel7_client.values()][0]
-        client_container_name = [key for key in rhel7_client.keys()][0]
-
-        self._host_location_update(client_container_name=client_container_name, loc=loc)
         wait_for(
-            lambda: org.name in execute(docker_execute_command,
-                                        client_container_id,
-                                        'subscription-manager identity',
-                                        host=self.docker_vm)[self.docker_vm],
-            timeout=100,
+            lambda: self.org.name in execute(docker_execute_command,
+                                             client_container_id,
+                                             'subscription-manager identity',
+                                             host=self.docker_vm)[self.docker_vm],
+            timeout=500,
```

Comment:
Any reason to increase directly from 100 to 500. Its a huge boost :)

Reply:
@jyejare I have noticed that sometime due to highload on docker system (subscription-manager taking time) hence increased it to avoid false fail.
also should impact much here .

Reply:
May be you can come up to 300. 500 is little higher.